### DISCUSSION TITLE
Syntax for figures

### DISCUSSION THREAD

**Discussion by mjoork** [jgm/djot#268] [Ideas]
In short, I think a syntax for the below construction would be awesome.

<pre><code>``` =html
&lt;figure&gt;
```

{% Figure content in djot %}

``` =html
&lt;figcaption&gt;
```

{% Figure caption in djot %}

``` =html
&lt;/figcaption&gt;&lt;/figure&gt;
```
</code></pre>

~~I think it's possible to reuse djot's syntax for table caption for this. Might be linked to [specially marked divs/blocks discussion](https://github.com/jgm/djot/discussions/197), but figures don't seem special to me.~~

Figures behave just like tables, except the figure's content is a direct child of the figure alongside the figure caption.

**Comment by jgm** [comment]
Agreed.  There is already an issue for this (#31).

**Comment by mjoork** [comment]
I overlooked it in issues, sorry for making a duplicate! I hope djot will succeed in it's mission, because it's very good!
